client/components/deposits-overview/index.tsx

```diff
@@ -25,11 +27,12 @@ interface OverviewProps {
 const DepositsOverviewDetails: React.FunctionComponent< OverviewProps > = (
 	props
 ) => {
+	const { isLoading }: OverviewProps = props;
```

Comment:
Nit: consider destructuring props directly in the function params to simplify.
```suggestion
const DepositsOverviewDetails: React.FunctionComponent< OverviewProps > = ( {
	isLoading,
} ) => {
```

Reply:
I like it, thanks!